src/components/ActivityMap/ActivityStats/ActivityStats.tsx

```diff
@@ -0,0 +1,25 @@
+import { Table } from '@mui/joy';
+
+import { ActivityStatRow } from './ActivityStatRow';
+import { useActivityStatsRows } from './useActivityStatsRows';
+
+export function ActivityStats() {
+  const { columns, statsRows } = useActivityStatsRows();
+
+  return (
+    <Table hoverRow>
+      <thead>
+        <tr>
+          {columns.map((c) => (
+            <th key={c}>{c}</th>
+          ))}
+        </tr>
+      </thead>
+      <tbody>
+        {statsRows.map((statsRow) => (
+          <ActivityStatRow key={statsRow.label} {...statsRow} />
+        ))}
+      </tbody>
+    </Table>
```

Comment:
This is the meat of the PR -- render a MUI Joy table, which renders `ActivityStatRow`, which themselves contain all the logic to render Strava data, Garmin data, or both based on which transform functions they're passed.